sr_utilities/nodes/temperature_monitor.py

```diff
@@ -34,11 +33,11 @@
 WARM = 55
 
 
-class Joint(object):
+class Joint:
     def __init__(self, screen, joint_name, x, y):
         self.screen = screen
-        self.x = x
-        self.y = y
+        self.x = x  # pylint: disable=C0103
+        self.y = y  # pylint: disable=C0103
```

Comment:
I guess the disabling can be removed with this to be merged https://github.com/shadow-robot/sr-build-tools/pull/662?

Reply:
yes it can

Reply:
it's merged now so removed this disable